File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioDtoLayoutLocmap.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnMapRenderingMode;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoLayoutLocmap extends StudioDtoLayoutGrid
{
  @Nullable
  public StudioDtoLayoutLocmapLineStroke lineStroke;

  @Nullable
  public StudioDtoLayoutLocmapPin liveLocationPin;

  @Nullable
  public MetaIdField locationFieldId;

  @Nullable
  public StudioDtoLayoutLocmapPin mapPin;

  @Nullable
  public EnumDefnMapRenderingMode renderingMode;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutLocmap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnMapRenderingMode
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.dto.StudioDtoLayoutGrid
import com.neome.api.meta.base.dto.StudioDtoLayoutLocmapLineStroke
import com.neome.api.meta.base.dto.StudioDtoLayoutLocmapPin

interface StudioDtoLayoutLocmap : StudioDtoLayoutGrid
{
  val lineStroke: StudioDtoLayoutLocmapLineStroke?
  val liveLocationPin: StudioDtoLayoutLocmapPin?
  val locationFieldId: MetaIdField?
  val mapPin: StudioDtoLayoutLocmapPin?
  val renderingMode: EnumDefnMapRenderingMode?
}